src/components/offerPage/index.jsx

```diff
@@ -0,0 +1,347 @@
+    // Can not select days before today
+    return current && current < moment().startOf("day");
+  }
+
+  const onFinish = (values) => {
+    createOffer(values, clickedLatLng, history);
+  };
+
+  const onFinishFailed = (values) => {
+    errorMessage(
+      offerErrorMessages.CREATE_OFFER_FAILED,
+      offerErrorMessages.ENTER_ALL_INPUTS
+    );
+  };
+
+  return isLoaded ? (
+    <>
+      <Header />
+      <div className="createOfferBody">
+        <h1>Create offer</h1>
+        <Form
+          form={form}
+          labelCol={{ span: 8 }}
+          wrapperCol={{ span: 16 }}
+          initialValues={{ remember: true }}
+          onFinish={onFinish}
+          onFinishFailed={onFinishFailed}
+          scrollToFirstError
+          className="formOffer"
+        >
+          <div className="leftFormBlock">
+            <Form.Item
+              name="address"
+              rules={[
+                {
+                  type: "string",
+                  message: inputValidationErrors.EMPTY_ADDRESS_MESSAGE,
+                },
+                {
+                  required: true,
+                  message: inputValidationErrors.EMPTY_ADDRESS_MESSAGE,
+                },
+              ]}
+            >
+              <PlacesAutocomplete
+                value={address}
+                onSelect={handleSelect}
+              >
+                {({
+                  getInputProps,
+                  suggestions,
+                  getSuggestionItemProps,
+                  loading,
+                }) => (
+                  <div>
+                    <Input
+                      {...getInputProps({ placeholder: "Enter your address" })}
+                    />
+
+                    <div className="description-list">
+                      {loading ? <div>...loading</div> : null}
+
+                      {suggestions.map((suggestion) => {
+                        return (
+                          <div
+                            {...getSuggestionItemProps(suggestion)}
+                            className="description-item"
+                          >
+                            {suggestion.description}
+                          </div>
+                        );
+                      })}
+                    </div>
+                  </div>
+                )}
+              </PlacesAutocomplete>
+            </Form.Item>
+            <Form.Item
+              name="settlement"
+              rules={[
+                {
+                  type: "string",
+                  message: inputValidationErrors.EMPTY_SETTLEMENT_MESSAGE,
+                },
+                {
+                  required: true,
+                  message: inputValidationErrors.EMPTY_SETTLEMENT_MESSAGE,
+                },
+              ]}
+            >
+              <Input name="settlement" placeholder="Enter your settlement" />
+            </Form.Item>
+            <Form.Item
+              name="region"
+              rules={[
```

Comment:
I can enter anything in this line, are you sure this is a good validation and that it matches the one in the service?